components/scream/src/control/tests/sc_tests.cpp

```diff
@@ -274,6 +274,12 @@ TEST_CASE ("recreate_mct_coupling")
   FID pseudo_density_id  ("pseudo_density",  scalar3d_layout, Pa,     grid_name);
   FID qv_id              ("qv",              scalar3d_layout, nondim, grid_name);
   FID precip_liq_surf_id ("precip_liq_surf", scalar2d_layout, m/s,    grid_name);
+  FID sfc_flux_dir_nir_id("sfc_flux_dir_nir", scalar2d_layout, W/(m*m), grid_name);
```

Comment:
So these units look great, but we named them a little differently in `atmosphere_radiation.cpp`, which @AaronDonahue pointed out is a bug (well, misleading, does not affect answers) and we need to fix it. For this PR, is there any harm in the face that here `sfc_flux_dir_vis` has units of `W/(m*m)` but in `atmosphere_radiation.cpp` we mistakenly labeled these with `Wm2`, which has the string "Wm2" for some reason?

Reply:
Isn't the actual units for "Wm2" in `atmosphere_radiation.cpp` actually `W/m/m`, so they are the same?

Either way, this test doesn't use the units at all, or interact with the AD, it just creates random field manager (that _just so happen_ to correspond to the AD fields), so it wouldn't matter if all the units were incorrect. But also, ideally they would be consistent.

Reply:
Yes, but the "string" is wrong in `atmosphere_radiation.cpp`. I for some reason set it to "Wm2". I could make the change in my pending PR to set these to "W/(m*m)" for consistency?

Reply:
This is currently what we have in `atmosphere_radiation`:
```
  auto Wm2 = W / m / m;
  Wm2.set_string("Wm2");
```

Reply:
Ahh I see. Yeah, that would be fine.

Reply:
Also just noticed there's no string representation here, so I think this comment is irrelevant.